src/LaunchDarkly.ServerSdk/LaunchDarkly.ServerSdk.csproj

```diff
@@ -38,6 +38,10 @@
   </PropertyGroup>
 
   <ItemGroup>
+    <PackageReference Include="IsExternalInit" Version="1.0.2">
```

Comment:
This package is compile time only.  Injects the falling class into your build:

[IsExternalInit](https://source.dot.net/#System.Private.CoreLib/IsExternalInit.cs,7c39b2fb54deae81):
```cs
using System.ComponentModel;

namespace System.Runtime.CompilerServices
{
    [EditorBrowsable(EditorBrowsableState.Never)]
    internal static class IsExternalInit {
    }
}
```

But it's a package dependency:

![image](https://user-images.githubusercontent.com/12101120/152626070-45402c62-eff8-4415-925a-3030ff47a57d.png)